scripts/solr_updater.py

```diff
@@ -30,7 +30,7 @@
 args: dict = {}
 
 
-def read_state_file(path, initial_state: str = None):
+def read_state_file(path, initial_state: Optional[str] = None):
```

Comment:
Why this change?

Reply:
This was an older change, since undone, that I may have inadvertently re-pushed.